.github/workflows/artifacts-build.yml

```diff
@@ -147,6 +147,7 @@ jobs:
     - name: Init
       run: |
         cd ffi/diplomat/js/examples/wasm-demo
+        npm i
```

Comment:
Issue: `npm ci` is better because it doesn't rebuild package-lock.json. Do we need `npm i` for it to work? Seems odd.

Reply:
Yes.